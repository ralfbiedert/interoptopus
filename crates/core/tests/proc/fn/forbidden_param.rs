use interoptopus::ffi_function;

#[ffi_function]
#[allow(unused_variables)]
fn foo(new: u8) {}

fn main() {}
