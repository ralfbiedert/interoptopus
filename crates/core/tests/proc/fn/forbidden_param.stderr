error: Using the name 'new' can cause conflicts in generated code.
 --> tests/proc/fn/forbidden_param.rs:5:8
  |
5 | fn foo(new: u8) {}
  |        ^^^
